project/plugins.sbt

```diff
@@ -16,7 +16,7 @@ resolvers ++= Seq(
   Resolver.url("bintray-sbt-plugin-releases", url("https://dl.bintray.com/content/sbt/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
 )
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.3")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.5")
```

Comment:
What are the implications of upgrading `Play`? Is it absolutely necessary for this PR to work?

Reply:
Just tends to be good practice on the CAPI team to have minor updates to Play frequently! Can remove if you'd like.

Reply:
Up to you. I haven't look at the changelog. 
I only want to avoid a situation where a problem cause by the new version of Play results in your changes being reverted at the same time.